web_responsive/static/src/css/web_responsive.scss

```
@@ -181,23 +179,32 @@ $chatter_zone_width: 35%;
 
         @include media-breakpoint-up(lg) {
             padding: {
-                left: 20vw;
-                right: 20vw;
+                left: calc((100vw - 850px) / 2);
+                right: calc((100vw - 850px) / 2);
             }
         }
 
         .o_app {
             align-items: center;
             display: flex;
+            padding: 10px 0;
+            border-radius: 4px;
             flex-direction: column;
             justify-content: flex-start;
             background: none;
+            transition: 300ms ease;
+            transition-property: background-color;
+
             img {
                 box-shadow: none;
                 transition: 300ms ease;
                 transition-property: box-shadow, transform;
             }
 
+            &:focus {
+                background-color: rgba(white, 0.05);
```

The bootstrap theme you use includes a definition of what is "white" and "black", a few shades of gray and all the primary, secondary, complementary colors. It's better to use those, so your addon looks good if there's any theme change.

```suggestion
                background-color: rgba(gray('white'), 0.05);
```

---

Thanks for the clarification!
Yes, I was aware of the theme-colors, but wasn't aware of the white/black for this case. Even in odoo source code there are lots of cases where the hardcoded keyword black/white is used.  

https://github.com/odoo/odoo/search?l=SCSS&q=white

---

> Even in odoo source code there are lots of cases where the hardcoded keyword black/white is used

:grin: I know, but well... It's odoo, not OCA :stuck_out_tongue_winking_eye: